src/Schema/Builder.php

```diff
@@ -225,9 +228,11 @@ public function getColumns($table)
 
     public function getIndexes($table)
     {
-        $indexes = $this->connection->getMongoDB()->selectCollection($table)->listIndexes();
-
+        $collection = $this->connection->getMongoDB()->selectCollection($table);
+        assert($collection instanceof Collection);
         $indexList = [];
+
+        $indexes = $collection->listIndexes();
```

Comment:
This is outside the diff but I think it's worth mentioning since I see you use an `===` to check for `['_id' => 1]`. If there is ever an issue with that, you may need to relax the comparison on the numeric value. IIRC, mongos used to sometimes return command results as `1.0` instead of `1` and I _think_ there may be similar inconsistencies for index definitions.

That said, it's probably fine to just wait and see if this is ever reported before going out of your way to address it now. It's quite possible that it's a non-issue on recent server versions.